docs/client-concepts/high-level/indexing/ingest-nodes.asciidoc

```diff
@@ -0,0 +1,42 @@
+:ref_current: https://www.elastic.co/guide/en/elasticsearch/reference/6.x
+
+:github: https://github.com/elastic/elasticsearch-net
+
+:nuget: https://www.nuget.org/packages
+
+////
+IMPORTANT NOTE
+==============
+This file has been generated from https://github.com/elastic/elasticsearch-net/tree/6.x/src/Tests/Tests/ClientConcepts/HighLevel/Indexing/IngestNodes.doc.cs. 
+If you wish to submit a PR for any spelling mistakes, typos or grammatical errors for this file,
+please modify the original csharp file found at the link and submit the PR with that change. Thanks!
+////
+
+[[ingest-node]]
+=== Ingest Node
+
+Elasticsearch will automatically re-route index requests to ingest nodes,
+however with some careful consideration you can optimise this path.
+
+==== Custom indexing client
+
+Since Elasticsearch will automatically reroute ingest requests to ingest nodes, you don't have to specify or configure any routing
+information. However, if you're doing heavy ingestion and have dedicated ingest nodes, it makes sense to send index requests to
+these nodes directly, to avoid any extra hops in the cluster.
+
+The simplest way to achieve this with any Elasticsearch client is to create a dedicated "indexing" client
+instance, and use it for indexing requests:
+
+[source,csharp]
+----
+var pool = new StaticConnectionPool(new [] <1>
+{
+    new Uri("http://ingestnode1:9200"),
+    new Uri("http://ingestnode2:9200"),
+    new Uri("http://ingestnode3:9200")
+});
+var settings = new ConnectionSettings(pool);
+var indexingClient = new ElasticClient(settings);
```

Comment:
Might be a nice opportunity here to show `SniffingConnectionPool` in conjunction with `NodePredicate()` too, to send index requests only to Nodes that have the ingest role

Reply:
Good idea, will add another example.